### ISSUE TITLE
[PWA] Problème de CSS sur le bouton d'accès au menu

### ISSUE THREAD

**Issue by bruhnild** [GeotrekCE/Geotrek-rando-v3#1062]
En dessous de 343 px (responsivité) de largeur d'écran, le menu en haut à droite de la page d'accueil n'est pas accessible (voir gig ci dessous)

![Peek 05-03-2024 11-18](https://github.com/GeotrekCE/Geotrek-rando-v3/assets/26433045/c2aaf817-08b5-4f7f-9cd5-c00b2eb0e736)

Cela est particulièrement gênant pour l'utilisateur qui souhaite acéder à ses contenus hors ligne.

**Comment by babastienne** [comment]
Non, ce n'est pas un bug du produit, c'est lié à la custo du territoire.

Exemple sur les écrins avec une largeur d'écran de 240px ça marche très bien : 

![image](https://github.com/GeotrekCE/Geotrek-rando-v3/assets/16348048/c73965f5-a26a-4bca-a84a-b08918df1b80)

**Comment by dtrucs** [comment]
Merci @bruhnild d'avoir remonté ce bug.
Effectivement @babastienne a raison c'est lié à la custo

**Comment by JonathanALLAIN** [comment]
Merci bcp pour vos investigations sur ce bugg et correctif.
